docs/main/user-guides/replicated-storage-user-guide/rs-deployment.md

```diff
@@ -0,0 +1,244 @@
+
+## Define the PVC
+
+Use `kubectl` to create a PVC based on a StorageClass that you created in the [previous stage](../replicated-storage-user-guide/rs-configuration.md#create-replicated-storageclasss). In the example shown below, we will consider that StorageClass to have been named "mayastor-1". Replace the value of the field "storageClassName" with the name of your own Replicated Storage-based StorageClass.
+
+For the purposes of this quickstart guide, it is suggested to name the PVC "ms-volume-claim", as this is what will be illustrated in the example steps which follow.
+
+**Command**
+
+```
+cat <<EOF | kubectl create -f -
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ms-volume-claim
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: mayastor-1
+EOF
+```
+
+**YAML**
+
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ms-volume-claim
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: INSERT_YOUR_STORAGECLASS_NAME_HERE
+```
+
+If you used the storage class example from previous stage, then volume binding mode is set to `WaitForFirstConsumer`. That means, that the volume won't be created until there is an application using the volume. We will go ahead and create the application pod and then check all resources that should have been created as part of that in kubernetes.
+
+## Deploy the FIO Test Pod
+
+The Replicated Storage CSI driver will cause the application pod and the corresponding Replicated Storage volume's NVMe target/controller ("Nexus") to be scheduled on the _same_ Replicated Storage Node, in order to assist with restoration of volume and application availabilty in the event of a storage node failure.
+
+:::warning
+In this version, applications using PVs provisioned by Replicated Storage can only be successfully scheduled on Replicated Storage Nodes. This behaviour is controlled by the `local:` parameter of the corresponding StorageClass, which by default is set to a value of `true`. Therefore, this is the only supported value for this release - setting a non-local configuration may cause scheduling of the application pod to fail, as the PV cannot be mounted to a worker node other than a MSN. This behaviour will change in a future release.
+:::
+
+**Command \(GitHub Latest\)**
+
+```
+kubectl apply -f https://raw.githubusercontent.com/openebs/Mayastor/v1.0.2/deploy/fio.yaml
```

Comment:
Should we use a later release yaml? @Abhinandan-Purkait ?

Reply:
This hasn't really changed. We have removed this in 2.x altogether. The latest that we have is 1.0.9 https://raw.githubusercontent.com/openebs/mayastor/v1.0.9/deploy/fio.yaml

Reply:
So, can we replace the link with the newer one (https://raw.githubusercontent.com/openebs/mayastor/v1.0.9/deploy/fio.yaml)?